Assets/Scripts/ArmScreen.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ArmScreen : MonoBehaviour {
+    private readonly Color[] colors = new Color[2];
+    private float scaleX, scaleY, scaleZ;
+    private Transform trans;
+    private Text text;
+    private RawImage image;
+    [SerializeField]
+    private Camera cam = null;
+    // Start is called before the first frame update
+    void Start() {
+        text = GetComponentInChildren<Text>();
+        image = GetComponent<RawImage>();
+        trans = GetComponent<Transform>();
+        scaleX = trans.localScale.x;
+        scaleY = trans.localScale.y;
+        scaleZ = trans.localScale.z;
+    }
+
+    // Update is called once per frame
+    void Update() {
+        if (Vector3.Dot(transform.right, cam.transform.up) > 0) {
+            trans.localScale = new Vector3(-scaleX, -scaleY, scaleZ);
+        } else {
+            trans.localScale = new Vector3(scaleX, scaleY, scaleZ);
+        }
+        text.text = "Balls: " + BallManager.Instance.playerBallQueues[0].childCount.ToString();
+        float a = Vector3.Dot(transform.forward, cam.transform.forward) > 0.5f
+            && !(Mathf.Abs(Vector3.Dot(transform.right, cam.transform.up)) < 0.8f) ? Vector3.Dot(transform.forward, cam.transform.forward) : 0;
+        ChangeTransparency(a);
+    }
+
+    private void ChangeTransparency(float alpha) {
```

Comment:
We shod prob move this to a commons soon
Something like `public void SetColor (Image image, Color c, float alpha)` or `public Color SetColor (Color c, float alpha)`

Reply:
yea `public Color SetTransparency(Color c, float alpha)` sounds about right

Reply:
Its not really setting transparency if you can also change the color as well depending on what color u pass in

Reply:
but considering it's taking an `alpha` the method signature kinda suggests it should be used for setting transparency, maybe you want `public Color SetColor(Color c)`?

Reply:
Hmm, true.